apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -576,6 +619,43 @@ const UniversitySearch: Screen<UniversitySearchProps> = ({
     }
   }
 
+  const handleUserInput = (value: string) => {
+    setSelectedPage(1)
+    searchTermHasBeenInitialized.current = true
+    setQuery(value)
+
+    const currentQueryParams = router.query
+    const updatedQueryParams = {
```

Comment:
Perhaps check here if the value is empty then we might want to just stop having the search param in the url.

This is handled by the usequerystate library which I've wanted to try out more: https://www.npmjs.com/package/next-usequerystate

It's already in the package.json if you want to try it out

Reply:
Updated to remove empty search param. Will check the library another time.